### DISCUSSION TITLE
Solectrus mit FOX ESS und Anatel USB Dongle

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#2163] [General]
Gibt es schon Jemanden der Solectrus mit einem FOX Wechselrichter mit Anatel USB Dongle am laufen hat ?
Suche für meinen Bruder nach einer Möglichkeit Solectrus mit seiner PV zu laufen zu bringen.
Bin für jede Hilfe dankbar.

**Comment by ledermann** [comment]
Da es [seit Kurzem](https://solectrus.de/blog/2023-06-01-mqtt-anbindung/) den MQTT-Collector für SOLECTRUS gibt, wäre das ein Ansatz:
https://github.com/solectrus/mqtt-collector

Allerdings erfordert dies einen MQTT-Broker und den Einsatz weiterer Software. Wenn der Wechselrichter z.B. von [ioBroker](https://www.iobroker.net/) oder [Home Assistant](https://www.home-assistant.io/) oder [evcc](https://evcc.io/) unterstützt wird, könnte das funktionieren. Die Einrichtung wird dadurch allerdings deutlich komplexer.

**Comment by Zeppelin-PV** [comment]
wie Du weißt, Georg, bin ich nur in der Lage mit Copy & Paste etwas zum Laufen zu bringen
d.h. ich bräuchte Jemanden der schon Solectrus mit FOX ESS am laufen hat und eine sehr gute Beschreibung how to... dazu liefert.
so wie Deine Beschreibung für Solectrus mit SENEC auf Raspberry, die ist top !!!
Vielleicht meldet sich ja jemand ...
